.github/actions/render-protocol-pdf/action.yml

```diff
@@ -0,0 +1,6 @@
+name: Render Zcash Protocol Specification
+description: GitHub Action to compile Zcash Protocol Specification LaTeX documents
+author: Deirdre Connolly
+runs:
+  using: docker
+  image: Dockerfile
```

Comment:
How does this work? If I were to guess, the `docker` being used here automatically runs `make`? Document the process.

Reply:
This is the Dockerfile in the root of the zips repo, symlinked into a more convenient place for the use of the action

Reply:
Added a comment